### PR TITLE
Feat/solarsteein telecom

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-5493
| License          | BSD 3-Clause

## Description

- [x] https://github.com/ovh/manager/pull/3388
- [x] https://github.com/ovh/manager/pull/3393
- [x] https://github.com/ovh/manager/pull/3398
- [x] https://github.com/ovh/manager/pull/3400
- [x] https://github.com/ovh/manager/pull/3401
- [x] https://github.com/ovh/manager/pull/3442
- [x] https://github.com/ovh/manager/pull/3452
- [x] https://github.com/ovh/manager/pull/3453
- [x] Extract from fc2c2af6f6e0397963fe8bca9bcfbf00608f21aa
- [x] https://github.com/ovh/manager/pull/3781
- [x] https://github.com/ovh/manager/pull/3991

## Related

- [x] TRANS-32189

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#15910.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report test#16144.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Comment by ovh-cds** [comment]
CDS Report test#16773.0 ✘  
* Stage 1
  * lint ✘  
  * SemVer ✔

**Review by antleblanc — DISMISSED** [review]

File: packages/manager/apps/freefax/src/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
         <title>FreeFax App</title>
```
Comment (mohammed-zahaf) [inline packages/manager/apps/freefax/src/index.html:4]: So we need translation ?

Comment (marie-j) [reply packages/manager/apps/freefax/src/index.html:4]: App is not currently available to customer so it is not an issue the title is not translated yet. Also I think it is out of scope of this PR as main point is to add the breadcrumb

**Review by antleblanc — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Pull request merged**